cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help

```diff
@@ -1,52 +1,57 @@
 NAME
-       Ballerina build - Builds Ballerina module(s) and generates executable
-       outputs.
+       ballerina build - build Ballerina sources and generate executables.
 
 SYNOPSIS
        ballerina build [-o <output>] [--sourceroot] [--off-line] [--skip-tests]
                        [--skip-lock] {<ballerina-file | module-name> | -a | --all}
 
 
 DESCRIPTION
-       Builds Ballerina module(s)/file and produces an executable jar file(s).
+       Build Ballerina module(s)/file and produces an executable jar file(s).
 
        Building a Ballerina project or a specific module in a project the
-       executable .jar files will be created in <project-root>/target/bin directory.
+       executable .jar files will be created in <project-root>/target/bin
+       directory.
 
-       Building a single Ballerina file will create an executable .jar file in the
-       current directory. The name of the executable file will be
+       Building a single Ballerina file will create an executable .jar file in
+       the current directory. The name of the executable file will be
        <ballerina-file-name>.jar.
 
-       If the output file is specified with the -o flag, the output
-       will be written to the given output file name. The -o flag will only work
-       for single files.
+       If the output file is specified with the -o flag, the output will be
+       written to the given output file name. The -o flag will only work for
+       single files.
 
 
 OPTIONS
-       -o <output>
-           Writes output to the given file. The provided output
-           filename may or may not contain the ‘.jar’ extension.
+       -o <output>, --output <output>
+           Write output to the given file. The provided output filename may or
+           may not contain the ‘.jar’ extension.
 
        --sourceroot <path>
-           Provides the path to be taken as the root of the source.
-           Source will be looked up relative to the given source root
-           path.
+           Provide the path to be taken as the root of the source. Source will
+           be looked up relative to the given source root path.
 
        -c, --compile
            Compile the source without generating executable(s).
 
        -a, --all
-           Build or compile all the modules of a project.
+           Build or compile all the modules within a project.
 
        --off-line
-           Builds offline without downloading dependencies.
+           Build offline without downloading dependencies.
 
        --skip-tests
-           Skips test compilation and execution.
+           Skip test compilation and execution.
 
        --skip-lock
            Skip using the lock file to resolve dependencies.
 
+       --experimental
+           Enable experimental language features.
+
+       --config
```

Comment:
There seems to be a new config format https://github.com/ballerina-platform/ballerina-lang/pull/18708/files#diff-fd96dc1ef9c62010f9406cc6373ea3f4R50

Reply:
This is a CLI option expects by the build command.
Looks like the old style config passing has not being removed form the build command.
Shall we fix it in a separate PR?

Reply:
+1